Source/reactions/Castro_react.cpp

```diff
@@ -843,11 +845,7 @@ Castro::react_state(Real time, Real dt)
 
     }
 
-    if (burn_success) {
-        return true;
-    } else {
-        return false;
-    }
+    return burn_success == 1;
```

Comment:
Now that the return type is int, can we just return burn_success?

Reply:
yes

Reply:
updated